WebService/appsettings.ini

```diff
@@ -45,6 +45,8 @@ twin_read_write_enabled = "${?PCS_TWIN_READ_WRITE_ENABLED}"
 # Format: DefaultEndpointsProtocol=https;AccountName=_____;AccountKey=_____;EndpointSuffix=core.windows.net
 iothub_import_storage_account_connstring = "${PCS_AZURE_STORAGE_ACCOUNT}"
 
+# User agent string identifying the application to Azure IoT hub
```

Comment:
Please fix grammar: "The user-agent string is used to identify the application to the Azure IoT Hub".

Reply:
done